NEMO.md

```diff
@@ -0,0 +1,268 @@
+```
+
+2. Create the arch files:
+
+`vim arch/arch-AARCH64_LINUX.env`
+
+```shell
+#HDF5安装目录
+export HDF5_INC_DIR="/path/to/hdf5/include"
+export HDF5_LIB_DIR="/path/to/hdf5/lib"
+#NetCDF安装目录
+export NETCDF_INC_DIR="/path/to/netcdf/include"
+export NETCDF_LIB_DIR="/path/to/netcdf/lib"
+```
+
+`vim arch/arch-AARCH64_LINUX.fcm`
+
+```shell
+################################################################################
+###################        Projet xios - xmlioserver       #####################
+################################################################################
+
+%CCOMPILER      mpicc
+%FCOMPILER      mpif90
+%LINKER         mpif90  
+
+%BASE_CFLAGS    -ansi -w
+%PROD_CFLAGS    -O3 -DBOOST_DISABLE_ASSERTS
+%DEV_CFLAGS     -g -O2 
+%DEBUG_CFLAGS   -g 
+
+%BASE_FFLAGS    -D__NONE__ 
+%PROD_FFLAGS    -O3
+%DEV_FFLAGS     -g -O2
+%DEBUG_FFLAGS   -g 
+
+%BASE_INC       -D__NONE__
+%BASE_LD        -lstdc++
+
+%CPP            cpp
+%FPP            cpp -P
+%MAKE           gmake
+```
+
+`vim arch/arch-AARCH64_LINUX.path`
+
+```shell
+NETCDF_INCDIR="-I /path/to/netcdf/include"
+NETCDF_LIBDIR="-L /path/to/netcdf/lib"
+NETCDF_LIB="-lnetcdff -lnetcdf"
+
+MPI_INCDIR="-I /path/to/OpenMPI/include"
+MPI_LIBDIR="-L /path/to/OpenMPI/lib"
+MPI_LIB="-lmpi"
+
+HDF5_INCDIR="-I /path/to/hdf5/include"
+HDF5_LIBDIR="-L /path/to/hdf5/lib"
+HDF5_LIB="-lhdf5_hl -lhdf5  -lz"
+```
+
+3. Run the build script:
+
+```shell
+./make_xios --job 32 --full --arch AARCH64_HCC_LINUX -dev
+```
+
+
+
+## Compiling and Installing NEMO
+
+Run the following command to obtain the source code package:
+
+
+    svn co https://forge.ipsl.jussieu.fr/nemo/svn/NEMO/releases/r4.0/r4.0.6
+    cd r4.0.6
+
+Edit the arch files:
+
+`vim arch/arch-aarch64_linux_gnu.fcm`
+
+
+```shell
+%NCDF_HOME           /path/to/netcdf
+%HDF5_HOME           /path/to/hdf5/
+%XIOS_HOME           /path/to/xios-2.5
+
+%NCDF_INC            -I%NCDF_HOME/include -I%HDF5_HOME/include
+%NCDF_LIB            -L%HDF5_HOME/lib -L%NCDF_HOME/lib -lnetcdff -lnetcdf
+%XIOS_INC            -I%XIOS_HOME/inc
+%XIOS_LIB            -L%XIOS_HOME/lib -lxios
+
+%CPP	             cpp -Dkey_nosignedzero
+%FC	             mpif90 -c -cpp
+%FCFLAGS             -mcpu=native -fdefault-real-8 -fdefault-double-8 -Ofast -funroll-all-loops -fcray-pointer -ffree-line-length-none -g -ffree-form
+%FFLAGS              %FCFLAGS
+%LD                  mpif90
+%LDFLAGS             -lstdc++
+%FPPFLAGS            -P -C -traditional
+%AR                  ar
+%ARFLAGS             rs
+%MK                  gmake
+%USER_INC            %XIOS_INC %NCDF_INC
+%USER_LIB            %XIOS_LIB %NCDF_LIB -lm
+
+%CC                  cc
+%CFLAGS              -O0
+```
+
+Change to the `config` directory and run the build script:
+
+
+```shell
+cd config
+./makenemo -m aarch64_linux_gnu -r 'target_configure' -n 'your_configure'
+```
+
+Check whether the installation is successful:
+
+
+    ll your_configure/EXP00/nemo
```

Comment:
I guess `ll` is an alias in your system for `ls -la` or something similar.
I think it would be better to use the full command instead.

Reply:
done